net.retrodeck.retrodeck.yml

```diff
@@ -946,6 +946,21 @@ modules:
 
   # PRIMEHACK - END
 
+  # Ruffle - START
+
+  - name: ruffle
+    buildsystem: simple
+    build-commands:
+      - mkdir -p "${FLATPAK_DEST}/share/ruffle"
+      - tar -zxvf *.tar.gz -C "${FLATPAK_DEST}/share/ruffle"
+      - chmod +x "${FLATPAK_DEST}/share/ruffle/ruffle"
+    sources:
+      - type: file
+        url: RUFFLEURLPLACEHOLDER
```

Comment:
Usually we use this approach only if the software is not properly versioned, for example when there is a single release that is overwritten. 
In all the other cases we prefer to keep a static version and update it manually as it's better for controlling the software.
For example sometimes happens that the latest version is not the best one due to bugs or other stuff.

Reply:
Yes if we a pick  version makes the sed less complicated too :)

It was more I fancied seeing how to do this.

But agree pick a tested version for main release.